libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -282,39 +368,17 @@ static bool _createKeepAliveOperation( const IotMqttNetworkInfo_t * pNetworkInfo
     /* Network information is not used when MQTT packet serializers are disabled. */
     ( void ) pNetworkInfo;
 
-    /* Default PINGREQ serializer function. */
-    IotMqttError_t ( * serializePingreq )( uint8_t **,
-                                           size_t * ) = _IotMqtt_SerializePingreq;
-
     /* Set PINGREQ operation members. */
     pMqttConnection->pingreq.u.operation.type = IOT_MQTT_PINGREQ;
 
     /* Convert the keep-alive interval to milliseconds. */
     pMqttConnection->pingreq.u.operation.periodic.ping.keepAliveMs = keepAliveSeconds * 1000;
     pMqttConnection->pingreq.u.operation.periodic.ping.nextPeriodMs = keepAliveSeconds * 1000;
 
-    /* Choose a PINGREQ serializer function. */
-    #if IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1
-        if( pNetworkInfo->pMqttSerializer != NULL )
-        {
-            if( pNetworkInfo->pMqttSerializer->serialize.pingreq != NULL )
-            {
-                serializePingreq = pNetworkInfo->pMqttSerializer->serialize.pingreq;
-            }
-            else
-            {
-                EMPTY_ELSE_MARKER;
-            }
-        }
-        else
-        {
-            EMPTY_ELSE_MARKER;
-        }
-    #endif /* if IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1 */
-
     /* Generate a PINGREQ packet. */
-    serializeStatus = serializePingreq( &( pMqttConnection->pingreq.u.operation.pMqttPacket ),
-                                        &( pMqttConnection->pingreq.u.operation.packetSize ) );
+    serializeStatus = (*_getMqttPingreqSerializer( pMqttConnection->pSerializer ))(
```

Comment:
Nit: Function pointers are invoked without the `*`.

Reply:
Removed throughout